[2022-07-22 14:14:12,848] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: load_data.get_nameo scheduled__2022-07-06T02:00:00+00:00 [queued]>
[2022-07-22 14:14:12,850] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: load_data.get_nameo scheduled__2022-07-06T02:00:00+00:00 [queued]>
[2022-07-22 14:14:12,850] {taskinstance.py:1376} INFO - 
--------------------------------------------------------------------------------
[2022-07-22 14:14:12,850] {taskinstance.py:1377} INFO - Starting attempt 1 of 6
[2022-07-22 14:14:12,850] {taskinstance.py:1378} INFO - 
--------------------------------------------------------------------------------
[2022-07-22 14:14:12,859] {taskinstance.py:1397} INFO - Executing <Task(PythonOperator): get_nameo> on 2022-07-06 02:00:00+00:00
[2022-07-22 14:14:12,861] {standard_task_runner.py:52} INFO - Started process 20123 to run task
[2022-07-22 14:14:12,863] {standard_task_runner.py:79} INFO - Running: ['airflow', 'tasks', 'run', 'load_data', 'get_nameo', 'scheduled__2022-07-06T02:00:00+00:00', '--job-id', '33', '--raw', '--subdir', 'DAGS_FOLDER/create_dag_data_loader.py', '--cfg-path', '/tmp/tmp7bd_xn13', '--error-file', '/tmp/tmpafwddsh2']
[2022-07-22 14:14:12,863] {standard_task_runner.py:80} INFO - Job 33: Subtask get_nameo
[2022-07-22 14:14:12,873] {logging_mixin.py:115} WARNING - /home/hp/miniconda3/envs/airflow_env/lib/python3.8/site-packages/airflow/configuration.py:532 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2022-07-22 14:14:12,874] {logging_mixin.py:115} WARNING - /home/hp/miniconda3/envs/airflow_env/lib/python3.8/site-packages/airflow/configuration.py:532 DeprecationWarning: The sql_engine_encoding option in [core] has been moved to the sql_engine_encoding option in [database] - the old setting has been used, but please update your config.
[2022-07-22 14:14:12,886] {task_command.py:371} INFO - Running <TaskInstance: load_data.get_nameo scheduled__2022-07-06T02:00:00+00:00 [running]> on host hp-HP-Pavilion-Laptop-15-eg0xxx
[2022-07-22 14:14:12,910] {taskinstance.py:1589} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=tesfaye
AIRFLOW_CTX_DAG_ID=load_data
AIRFLOW_CTX_TASK_ID=get_nameo
AIRFLOW_CTX_EXECUTION_DATE=2022-07-06T02:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2022-07-06T02:00:00+00:00
[2022-07-22 14:14:12,911] {create_dag_data_loader.py:16} INFO - hello there this is my first python operator dag
[2022-07-22 14:14:12,911] {python.py:173} INFO - Done. Returned value was: None
[2022-07-22 14:14:12,914] {taskinstance.py:1415} INFO - Marking task as SUCCESS. dag_id=load_data, task_id=get_nameo, execution_date=20220706T020000, start_date=20220722T111412, end_date=20220722T111412
[2022-07-22 14:14:12,955] {local_task_job.py:156} INFO - Task exited with return code 0
[2022-07-22 14:14:12,974] {local_task_job.py:273} INFO - 0 downstream tasks scheduled from follow-on schedule check
